Frontend/CO.CDP.OrganisationApp/appsettings.json

```diff
@@ -16,6 +16,7 @@
   "FormsService": "",
   "DataSharingService": "",
   "EntityVerificationService": "",
+  "FtsService": "",
```

Comment:
@dpatel017 I just thought it would be good to configure this to something in `appsettings.Development.json`, so we'd see it working in local development.